### PR TITLE
advertisers section revamp

### PR DESCRIPTION
**Pull request by philburrrt**
Notes:
- Split up advertisers section, revised copy & images on all
- Upgraded Docusaurus to 2.0.0-beta.17
- Added plugin to expand images

### CODE REVIEW

File: about/welcome.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Banner spaces are represented as NFTs - ownership of the NFT gives the renter th
 
 ## Advertise
 
-Learn how to rent banner spaces to promote your business or brand [here](../guides/advertisers).
+Learn how to rent banner spaces to promote your business or brand [here](../guides/for-advertisers/bid).
```
Comment (limbofeather) [inline about/welcome.md:15]: "to promote your project, app or brand"

File: guides/for-advertisers/bid.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Bid on Timeslots
+
+Virtual space owners list prices for some amount of time on their banners. Advertisers can bid on these timeslots and, if approved by the owner, be set up to automatically occupy and pay for that period of time.
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:3]: "Virtual billboard owners"

File: guides/for-advertisers/bid.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+- [Bid on Timeslots](#bid-on-timeslots)
+  - [1. Find a Space](#1-find-a-space)
+  - [2. Bid on a timeslot](#2-bid-on-a-timeslot)
+  - [3. Submit your purchase](#3-submit-your-purchase)
+  - [4. Wait for owner approval](#4-wait-for-owner-approval)
+
+<center><div class="iframe" position="relative" padding-bottom="56.25%" padding-top="30px" height="0" overflow="hidden"><iframe width="90%" height="432" src="https://www.youtube.com/embed/lEM9-r4PiRQ" title="YouTube video player" frameborder="0" position="absolution" top="0" left="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div></center>
+
+## 1. Find a Space
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:15]: "Find a billboard"

File: guides/for-advertisers/bid.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+When connected to the blockchain, you can find all active an inactive listings in the [Billboard Market](https://app.zesty.market/).
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:17]: What do you mean by being "connected to the blockchain"?

File: guides/for-advertisers/bid.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+:::tip filter results
+You can filter by criteria such as **Format**, **Active Listings** or **Lowest Price**
+:::
+
+![](<../../.gitbook/assets/image (12).png>)
+
+
+## 2. Bid on a timeslot
+
+Once you've found a space, look for the **Auctions** tab towards the middle of the page.
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:30]: space --> billboard

File: guides/for-advertisers/bid.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+:::caution Having trouble?
+Sometimes the network can be slow. It might help to refresh and try again.
+
+Reach out to us on [Discord](https://discord.gg/4Jc3XhM5mp) if you need technical support!
+:::
+
+## 4. Wait for owner approval
+
+Space owners need to approve your bid before the purchase goes through. They will be able to see all of your Campaign's details before making a decision.
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:67]: "Billboard owners"

File: guides/for-advertisers/bid.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+Watch your approval status in the Space's **Auctions** tab. If they accept, your campaign will appear in their space for the time specified in the auction.
```
Comment (limbofeather) [inline guides/for-advertisers/bid.md:69]: Should leave a note here that they can type in `>notify` in the `#bot-commands` channel in the Zesty Discord to get a notification when something is approved. Pls confirm that this is the case @JSeam2

File: guides/for-advertisers/create-campaign.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
+# Create a Campaign
+
+Campaigns are your gateway to brand exposure in the metaverse. Setting one up is easy, here's what you'll need to do:
```
Comment (limbofeather) [inline guides/for-advertisers/create-campaign.md:3]: Nice, like the wording "brand exposure"

File: guides/for-advertisers/refunds.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Refunds
+
+You can get refunds on either **bids you want to cancel** or **Space owners failing to display banner**
```
Comment (limbofeather) [inline guides/for-advertisers/refunds.md:3]: space owners --> billboard owners

**Review by limbofeather — COMMENTED** [review]
Left a couple of comments to be addressed, then can review again. Thanks for your work @philburrrt!

File: about/welcome.md (hunk 1 above)
Comment (philburrrt) [reply about/welcome.md:15]: Fixed. 
For reference, I had only moved this page, not written any copy.

File: guides/for-advertisers/bid.md (hunk 2 above)
Comment (philburrrt) [reply guides/for-advertisers/bid.md:3]: fixed

File: guides/for-advertisers/bid.md (hunk 4 above)
Comment (philburrrt) [reply guides/for-advertisers/bid.md:17]: You need to connect to polygon in order for the billboard listings to show

File: guides/for-advertisers/bid.md (hunk 7 above)
Comment (philburrrt) [reply guides/for-advertisers/bid.md]: Added this, will remove if jseam says otherwise

**Comment by philburrrt** [comment]
@limbofeather addressed all comments

File: guides/for-advertisers/bid.md (hunk 4 above)
Comment (philburrrt) [reply guides/for-advertisers/bid.md:17]: Correction. I fixed a link in it**

**Comment by limbofeather** [comment]
Thanks for addressing all of the comments @philburrrt, looks good to merge!

**Review by limbofeather — APPROVED** [review]

**Pull request merged**